File: tests/fixtures/sample.rb
module Animals
  class Dog
    def bark
      "woof!"
    end

    def self.species
      "Canis familiaris"
    end

    private

    def secret
      "hidden"
    end
  end
end

class Config
  def initialize(name, value = 0)
    @name = name
    @value = value
  end

  def get_name
    @name
  end
end

def greet(name)
  "Hello, #{name}!"
end

MAX_SIZE = 1024

File: tests/fixtures/sample.py
import os
from typing import Optional

def greet(name: str) -> str:
    return f"Hello, {name}!"

def add(a: int, b: int) -> int:
    return a + b

MAX_SIZE = 1024

class Config:
    default_name = "untitled"

    def __init__(self, name: str, value: int = 0):
        self.name = name
        self.value = value

    def get_name(self) -> str:
        return self.name

    @staticmethod
    def create(name: str) -> "Config":
        return Config(name)

@decorator
def decorated(x: int) -> int:
    return x * 2
